Source/Core/Common/Logging/LogManager.cpp

```diff
@@ -10,15 +10,23 @@
 
 #include "Common/CommonPaths.h"
 #include "Common/FileUtil.h"
-#include "Common/IniFile.h"
 #include "Common/Logging/ConsoleListener.h"
 #include "Common/Logging/Log.h"
 #include "Common/Logging/LogManager.h"
 #include "Common/StringUtil.h"
 #include "Common/Timer.h"
+#include "Core/Config/Config.h"
 
 constexpr size_t MAX_MSGLEN = 1024;
 
+const Config::ConfigInfo<bool> LOGGER_WRITE_TO_FILE{
```

Comment:
Not sure what everyone thinks of this but we ended up putting the video config options under `Source/Core/Core/Config` in their own files.  Does it make sense to do the same thing with the logging options?  It'd be nice to have the options all in one place.